.vscode/settings.json

```diff
@@ -1,57 +1,49 @@
-    "python.testing.pytestEnabled": true,
-    "python.testing.unittestEnabled": false,
-    "python.testing.promptToConfigure": true,
-    "python.envFile": "${workspaceFolder}/.env",
-    "python.formatting.autopep8Args": [
-        "--max-line-length=110",
-        "--ignore=E402"
-    ],
-    "files.exclude": {
-        ".github": false,
-        "**/.pytest_cache": true,
-        "**/__pycache__**": true,
-        "web/node_modules": true,
-        "api/python_cache": true,
-        "api/.venv": true
-    },
-    "editor.codeActionsOnSave": {
-        "source.fixAll.eslint": true
-    },
-    "editor.formatOnSave": true,
-    "eslint.format.enable": true,
-    "eslint.validate": [
-        "javascript",
-        "typescript"
-    ],
-    "eslint.run": "onSave",
-    "[python]": {
-        "editor.rulers": [110]
-    },
-    "[javascript]": {
-        "editor.defaultFormatter": "esbenp.prettier-vscode",
-        "editor.tabSize": 2
-    },
-    "[typescript]": {
-        "editor.defaultFormatter": "esbenp.prettier-vscode",
-        "editor.tabSize": 2
-    },
-    "[typescriptreact]": {
-        "editor.defaultFormatter": "esbenp.prettier-vscode",
-        "editor.tabSize": 2
-    },
-    "[yaml]": {
-        "editor.defaultFormatter": "esbenp.prettier-vscode",
-        "editor.insertSpaces": true,
-        "editor.tabSize": 2,
-        "editor.formatOnSave": true
-    },
-    "[json]": {
-        "editor.defaultFormatter": "vscode.json-language-features"
-    },
-    "python.testing.pytestArgs": [
-        "api"
-    ],
-    "typescript.preferences.importModuleSpecifier": "non-relative"
-}
\ No newline at end of file
+  "python.defaultInterpreterPath": "${workspaceFolder}/api/.venv/bin/python",
+  "python.testing.pytestEnabled": true,
+  "python.testing.unittestEnabled": false,
+  "python.testing.promptToConfigure": true,
+  "python.envFile": "${workspaceFolder}/.env",
+  "python.formatting.autopep8Args": ["--max-line-length=120", "--ignore=E402"],
+  "files.exclude": {
+    ".github": false,
+    "**/.pytest_cache": true,
+    "**/__pycache__**": true,
+    "web/node_modules": true,
+    "api/python_cache": true,
+    "api/.venv": true
+  },
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true
+  },
+  "editor.formatOnSave": true,
+  "eslint.format.enable": true,
+  "eslint.validate": ["javascript", "typescript"],
+  "eslint.run": "onSave",
+  "[python]": {
+    "editor.rulers": [120]
+  },
+  "[javascript]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode",
+    "editor.tabSize": 2
+  },
+  "[typescript]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode",
+    "editor.tabSize": 2
+  },
+  "[typescriptreact]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode",
+    "editor.tabSize": 2
+  },
+  "[yaml]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode",
+    "editor.insertSpaces": true,
+    "editor.tabSize": 2,
+    "editor.formatOnSave": true
+  },
```

Comment:
This formats different for me, because I have user settings for jsonc that differs from yours:
```
"[jsonc]": {
    "editor.defaultFormatter": "vscode.json-language-features"
  },
```
Can you please add in config for jsonc  to this file?
Can we please standardize on json and use "vscode.json-language-features" for jsonc. I like it a bit more right now - but honestly could go for prettier as well if you have a preference.